client.qbs

```diff
@@ -133,4 +133,5 @@ CppApplication {
         ]
     }
 
+    cpp.cxxFlags: ["-std=c++11"]
```

Comment:
This would be nicer, but probably not yet supported by qbs 1.3.3:

http://doc-snapshot.qt-project.org/qbs/cpp-module.html#cxxlanguageversion